batteries.opam

```diff
@@ -19,7 +19,8 @@ build: [
 install: [make "install"]
 run-test: [make "test"]
 depends: [
-  "ocaml" {>= "4.00.0" & < "4.15.0"}
+  "ocaml" {>= "5.00.0"}
```

Comment:
Is this lower bound supposed to be temporary? Because if it isn't, then there's no point to any of the conditional preprocessing.

Reply:
What I wanted to say in the opam file is: if ocaml >= 5 then depend on camlp-streams.
But, I don't know how to say that exactly.

Reply:
@kit-ty-kate Dear Kate, do you know how to say this to opam: "if ocaml >= 5 then depend on camlp-streams" ?

Reply:
Maybe something like this:
```
  "ocaml" {>= "4.00.0"}
  ("ocaml" {< "5.0"} | "camlp-streams")
```
There are variations of this in some opam-repository packages at least.

Reply:
yeah something like that would work though i would encourage you to always use camlp-streams for simplicity sake. On OCaml < 5.0 the module is already a simple alias to `Stdlib.Stream`

Reply:
camlp-streams has ocaml lower bound 4.02.3, but batteries currently has 4.00.0. If we accept that minimum bump, then it'd be cleaner indeed.

Reply:
I went with @kit-ty-kate 's suggestion